src/main/java/CarRacing/domain/Position.java

```diff
@@ -0,0 +1,19 @@
+package CarRacing.domain;
+
+public class Position {
+    private static final int INITIAL_POSITION = 1;
+
+    private int position;
+
```

Comment:
이전에 VO로 래핑해보면 좋겠다는 피드백을 받아 Position이라는 클래스를 만드신 것 같아요
https://tecoble.techcourse.co.kr/post/2020-06-11-value-object/
링크 한 번 읽어보시면 VO를 어떻게 활용할 수 있는지, 불변을 어떻게 지키는지 도움이 될 것 같네요

```suggestion
public class Position {
    private static final int INITIAL_POSITION = 1;

    private final int position;

```

Reply:
Position이 불변이라는 가정하에
상태 변경은 새로운 객체를 만들어 반환하면 될 것 같아요
```java
public Position sum(int distance) {
        return new Position(this.distance + distance);
}
```

Reply:
추가로 Position과 CarName에 대한 테스트도 만들어 보면 좋을 것 같아요
- 예시
  - Position은 음수가 될 수 없다